department-of-veterans-affairs/caseflow
# Kamalam7/appeals 35472

Resolves https://jira.devops.va.gov/browse/APPEALS-35472

# Description
When we Edit and save the changes to CMP Information and General Information in the Review package page the status of Review package task will changed to in-progress and if we have any previous efolder upload failure task on the correspondence the error banner has to display on the Review package page.

## Acceptance Criteria

## Testing Plan

1. Login as a Mail user.
2. Click on Queue from the App selector.
3. In the Your cases page from the Switch views, select Correspondence cases.
4. From the Correspondence cases click on any Veteran Details.
5. In you made any changes to the CMP Information Edit modal and General Information table and save the changes the Review package task status should be in-progress.
6. And if the Correspondence have any efolderuploadfailed task, the error banner has to be displayed on the Review package page.


1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below


# Frontend

<img width="881" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110078646/ebbc6651-892d-41c2-8a11-002f85d9c23c">

## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
